### DISCUSSION TITLE
When are the CSharp 10 features / proposals moved into the `csharp-10.0` folder?

### DISCUSSION THREAD

**Discussion by bernd5** [dotnet/csharplang#4918] [Q&A] [answered]
If I look at https://github.com/dotnet/roslyn/blob/main/docs/Language%20Feature%20Status.md there seems to quite concrete set of features which make it into C#10 - but when are those moved?
And why is 
`file-scoped-namespaces.md` in the CSharp 10 folder? In the roslyn repo it is in `CSharp C# Next` section (btw. it seems to be fully implemented in roslyn, the branch is deleted and it was merged to main). Which document is right?

**Comment by CyrusNajmabadi** [comment]
File scoped namespaces should be in c# 10

**Comment by bernd5** [comment, marked as answer]
they are moved now
